Sources/StringExtractor/StringParser.swift

```diff
@@ -68,16 +74,17 @@ extension StringParser {
         }
 
         // Optional, precision information
-        Optionally(.anyOf("-+# 0"))
+        Optionally(.anyOf("-+"))
```

Comment:
I removed space because while it's valid for printf specifiers, it's confusing.. For example:

```swift
String(format: "50% off") // "500ff"
```

It's not really used for localized strings, so there is no need to factor it in/ Same with `0` and `#`